src/index.js

```diff
@@ -35,6 +35,20 @@ function unknown(e, log) {
   return new Response(body, { status });
 }
 
+/**
+ * Parse a boolean given as either a string or a boolean.
+ *
+ * @param {any} value value to parse
+ * @param {boolean} defaultValue default value to use if value is unset
+ * @returns true, false or the default value
+ */
+function parseBoolean(value, defaultValue) {
```

Comment:
interesting that you still have a 100% coverage without an explicit `parseBoolean` test.

Reply:
I have one test that passes `'false'` (as string) in `value`, one that passes `true` (as boolean) and all other take default values, so this probably accounts for all branches.